FAQ.md

```diff
@@ -0,0 +1,100 @@
+# Frequently Asked Questions
+
+An index of frequently asked questions regarding all things ECMA-262.
+
+# Process Questions
+
+##### What is the process for proposing a new feature?
+
+New features start life as a proposal to the [TC39](#what-is-a-tc39) committee and must be championed (or co-championed) by at least one member of the committee. Once the proposal is raised at a committee meeting, it will become a Stage 0 proposal and move along from there. For more details on how proposal stages work, check out the [proposal process document][proposal-process-document].
+
+If you would like to contribute, please check out [Contributing to ECMAScript](https://github.com/jeffmo/ecma262/blob/master/CONTRIBUTING.md).
```

Comment:
Change `jeffmo` to `tc39` in `https://github.com/jeffmo/ecma262/blob/master/CONTRIBUTING.md` or use the local path if possible (`CONTRIBUTING.md`)? :smile: